knowledge/technical_manual/instructlab/qna.yaml

```diff
@@ -0,0 +1,38 @@
+---
+created_by: makelinux
+domain: AI
+seed_examples:
+  - question: What is the main goal of the InstructLab project?
+    answer: The main goal of the InstructLab project is to leverage innovative
+      techniques for overcoming challenges in LLM training, using
+      taxonomy-based curation and synthetic data generation to enable
+      contributions to existing LLMs.
```

Comment:
```suggestion
    answer: |
      The main goal of the InstructLab project is to leverage innovative
      techniques for overcoming challenges in LLM training, using
      taxonomy-based curation and synthetic data generation to enable
      contributions to existing LLMs.
```

See https://yaml-multiline.info/ for info on using multi-line strings in yaml.

Your other answers will need similar fixes.

Reply:
Oh, BJ, that is a great link. I'm stealing it.

Reply:
@bjhargrave , why do suggest "literal style, indicated by a pipe (|)." and not "folded style, indicated by a right angle bracket (>). "?

Reply:
It retains the new lines which can be important for certain information like tables. So we generally suggest `|` as the default choice here. If your value is pure text where new lines are just whitespace, `>` is fine also.

Reply:
What you have is ok as a plain flow scalar.

Reply:
@bjhargrave , so leave it as it is or add `>-`? 

> To remove all newlines, strip them by putting a minus sign (-)